### PR TITLE
Upgrade minikube and K8S version

### PR DESCRIPTION
**Pull request by abbi-gaurav**
**Description**

- Use Minikube version       : 0.33.0
- Use Kubernetes version   : 1.11.5
- Update minikube.sh
- Update minikube.ps1
- Update ci build files
- Update details for local installation

**To Do**
- [x] Verify console UI access (minikube)
- [x] Verify tests pass with `minikube`
  - all tests passed except `test-core-core-ui-acceptance`
  - confirmed with @hardl that it is not an upgrading related problem
- [x] Verify [e2e scenario](https://github.com/kyma-incubator/examples/tree/master/kyma-local-e2e-scenario) locally
- [x] Verify RBAC 
- [x] Verify on windows installation @venturasr verified.
- [x] Update prow to use new version
- [x] Fix error statements in the installation run. (They are not causing any issues at present as it seems) See [comment](https://github.com/kyma-project/kyma/pull/2346#issuecomment-455176037)
- [x] install on GKE (follow up with farros installation)
  - run e2e tests against GKE installation
- [x] Verify with feature gates
  - [x] knative
  - [x] ~xoip (not required to verify as not yet in a working state)~
- [x] ensure all admission plugins are applied
- [x] Check impact on the installer code. Does the operator needs to be upgraded?
- [x] Finalize the minikube, kubectl and kubernetes version
  - Currently `0.33.0`, `1.11.0` and `1.11.5` are used. 
- [x] Impact on controllers/operators in Kyma
- [x] Any missing documentation

### CODE REVIEW

**Comment by abbi-gaurav** [comment]
Errors outputs in installation
```bash
E0117 13:57:54.053845   26631 status.go:137] Error executing status template: template: status:1:2: executing "status" at <.MinikubeStatus>: can't evaluate field MinikubeStatus in type cmd.Status
minikube is not currently running so the service cannot be accessed
Starting local Kubernetes v1.11.3 cluster...
Starting VM...
Getting VM IP address...
E0117 13:58:29.152774   26634 start.go:210] Error parsing version semver:  Version string empty
```

**Comment by abbi-gaurav** [comment]
Prow job for integration test has currently hard-coded k8s version (v0.28.2). Waiting for a change from the prow side to watch for changes in the file https://github.com/kyma-project/kyma/blob/master/prow/kyma-integration-on-debian/install-deps-debian.sh

This will enable the build job for verification to succeed.

As discussed with @suleymanakbas91, going further, we should consider making the tooling versions updatable via a file in Kyma repository so that they can be made a part of the PR.

File: installation/scripts/minikube.ps1 [hunk 1]
```diff
@@ -15,7 +15,7 @@ Write-Output @"
 "@
 
 function CheckIfMinikubeIsInitialized() {
-    $cmd = "minikube status --format '{{.MinikubeStatus}}'"
+    $cmd = "minikube status --format '{{.Host}}'"
```
Comment (venturasr) [inline installation/scripts/minikube.ps1]: This change also needs to be done in line 70

Comment (abbi-gaurav) [reply installation/scripts/minikube.ps1]: Thanks.

**Comment by abbi-gaurav** [comment]
waiting for the PR #2395 to be merged first

**Comment by abbi-gaurav** [comment]
See also https://github.com/kyma-project/kyma/issues/2276

**Pull request merged**